src/Plutus/Certification/Server.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE ImportQualifiedPost #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Why was this needed?

Reply:
https://github.com/input-output-hk/dapps-certification/blob/286f38211d3c761a24db1878fd890f6ffa149294/src/Plutus/Certification/Server.hs#L61

Reply:
done

Reply:
Ah no that was fine, just curious.